### ISSUE TITLE
Design.IO is updating templates in console, but not in browser

### ISSUE THREAD

**Issue by activestylus** [tower-archive/tower#36]
When I make any change to a template I can see the Design.IO console responding:

```
[Sat, 03 Mar 2012 12:06:47 GMT] INFO updated app/views/sites/new.coffee
info:  Name: design.io-watcher::wordup exec                                     Type: hook            Data: {"body":"contentFor \"title\", \"New Title\"\n\npa ... 
info:  Name: design.io-watcher::wordup-0 design.io-watcher::wordup-0::exec        Type: hook            Data: {"body":"contentFor \"title\", \"New Title\"\n\npa ... 
[Sat, 03 Mar 2012 12:06:48 GMT] INFO updated public/javascripts/app/views/templates.js
  Processing by SitesController#new as HTML
  Parameters:
{ format: null, action: 'new' }

  GET /sites/new
  response time: 42ms
  memory rss: 3.94mb
  memory vsize: NaNgb
  heap before: 16.09mb / 18.12mb
  heap after: 18.51mb / 22.22mb
```

However when I go to my browser and refresh, my changes are not visible. I have to reboot the tower server in order to update my views!

The app I am testing with can be found here: https://github.com/activestylus/towerjs-test

Though it is worth noting that I faced the same issue when pulling the towerjs.org homepage repo

**Comment by lancejpollard** [comment]
Updated `design.io/lib/client.coffee`, which compiles into `design.io.js` for the client.  Should work now, try it and let me know.  Either replace `vendor/javascripts/design.io.js` with [this](https://raw.github.com/viatropos/design.io/master/design.io.js), or when you generate a new app it will be downloading that anyway.

Make sure you start the global socket server with `design.io start` in some terminal window.  Then in any tower app (or any directory with the design.io node module installed running the watcher with `design.io watch`), you should get automatic refreshes when you view the page.  Try adding this to `app/client/stylesheets/application.styl` after you've loaded the default white page at http://localhost:3000:

``` css
html, body
  background: red
```

Hope that helps.

**Comment by fauzias** [comment]
@viatropos same issue happen here, and that also include the controller code changes, nothing will be reloaded in the browser (either automatic refresh nor manual refresh), and all i have to do was exiting the server.js and rerun it again, such a wasted time shall not happen

**Comment by activestylus** [comment]
Still not working. Here is what I did, starting from scratch
1. `npm install design.io -g`
2. `npm install tower -g`
3. `design.io start`
4. New terminal window
5. `tower new design-test && cd design-test`
6. `npm install -d`
7. `tower server -p 3000`

At this point the app is running. But no matter what changes I make, whether it's to view templates or application.styl, they are not reflected in the browser (even after refreshing) I manually updated design.io.js and no luck there either.

FWIW I got this working with plain old Express and a simple kckr install https://github.com/pthrasher/kckr
